src/test/java/com/hyperwallet/clientsdk/model/HyperwalletPayPalAccountTest.java

```diff
@@ -13,7 +13,7 @@ protected HyperwalletPayPalAccount createBaseModel() {
         payPalAccount
                 .token("test-token")
                 .type(HyperwalletTransferMethod.Type.PAYPAL_ACCOUNT)
```

Comment:
Do we have any separate ticket to change HyperwalletTransferMethod.Typeto HyperwalletPayPalAccount.Type?

Reply:
done

Reply:
Why this line is removed completely?